src/Azure.Functions.Cli/StaticResources/customHandler.json

```diff
@@ -0,0 +1,8 @@
+﻿{
```

Comment:
When you do `func init` with custom handler the config here would be added to host.json. If someone runs `func start` after that without making any changes, the function app would run into errors since the config is incomplete.

Instead we should add a wizard similar to node, where once you select node you further prompter to select language ` JavaScript or TypeScript` we should prompt for the required fields for the customer to input on the init command.

Reply:
Doing a func start even after doing func new, they would still run into errors unless they configured the right parameters, so not sure if it's worth adding a wizard like experience. I'll let @pragnagopa or @anthonychu chime in.